btcjson/chainsvrresults.go

```diff
@@ -108,6 +108,7 @@ type GetBlockVerboseTxResult struct {
 	VersionHex    string        `json:"versionHex"`
 	MerkleRoot    string        `json:"merkleroot"`
 	Tx            []TxRawResult `json:"tx,omitempty"`
+	RawTx         []TxRawResult `json:"rawtx,omitempty"` // Note: this field is always empty when verbose != 2.
```

Comment:
Can you also mention that the field is deprecated in Bitcoin Core, as of at least 0.16.0?